<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>516</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <property name="windowFilePath">
   <string>Настройки</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>Общее</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabView">
      <attribute name="title">
       <string>Вид</string>
      </attribute>
      <widget class="QSplitter" name="splitter_2">
       <property name="geometry">
        <rect>
         <x>22</x>
         <y>22</y>
         <width>251</width>
         <height>117</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QWidget" name="layoutWidget1">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="lineWidth">
           <property name="text">
            <string>Толщина линии</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="handleSize">
           <property name="text">
            <string>Размер узла</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="numberSize">
           <property name="text">
            <string>Размер нумерации</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget2">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QSpinBox" name="spinLineWidth">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>50</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinHandleSize">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>50</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinNumberSize">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>50</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tabColor">
      <attribute name="title">
       <string>Цвет</string>
      </attribute>
      <widget class="QSplitter" name="splitter">
       <property name="geometry">
        <rect>
         <x>22</x>
         <y>11</y>
         <width>331</width>
         <height>281</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QWidget" name="layoutWidget3">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="handleColor">
           <property name="text">
            <string>Цвет узла</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="handleSelectColor">
           <property name="text">
            <string>Цвет выбранного узла</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="numberColor">
           <property name="text">
            <string>Цвет нумерации</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="backgroundColor">
           <property name="text">
            <string>Фон нумерации</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="rectColor">
           <property name="text">
            <string>Цвет прямоугольника</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="circleColor">
           <property name="text">
            <string>Цвет круга</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="polylineColor">
           <property name="text">
            <string>Цвет полилинии</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lineColor">
           <property name="text">
            <string>Цвет линии</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="minimumSize">
            <size>
             <width>26</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>26</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 1px solid #777; 
background: #000000;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="minimumSize">
            <size>
             <width>26</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>26</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 1px solid #777; 
background: #000000;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="minimumSize">
            <size>
             <width>26</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>26</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 1px solid #777; 
background: #000000;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="minimumSize">
            <size>
             <width>26</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>26</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 1px solid #777; 
background: #000000;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_5">
           <property name="minimumSize">
            <size>
             <width>26</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>26</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 1px solid #777; 
background: #000000;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_6">
           <property name="minimumSize">
            <size>
             <width>26</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>26</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 1px solid #777; 
background: #000000;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_7">
           <property name="minimumSize">
            <size>
             <width>26</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>26</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 1px solid #777; 
background: #000000;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_8">
           <property name="minimumSize">
            <size>
             <width>26</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>26</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 1px solid #777; 
background: #000000;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tabWorkdir">
      <attribute name="title">
       <string>Рабочая директория</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabPolyline">
      <attribute name="title">
       <string>Замыкание полилинии</string>
      </attribute>
      <widget class="QWidget" name="polylineLayoutHost">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>501</width>
         <height>138</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QRadioButton" name="doubleClickLMB">
          <property name="text">
           <string>По двойному клику ЛКМ</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="LMBZeroPoint">
          <property name="text">
           <string>По одинарному клику ЛКМ на нулевой точке</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="holdingCtrl">
          <property name="text">
           <string>Поставить последнюю точку с зажатым Ctrl</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="keyC">
          <property name="text">
           <string>По клавише C</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tabLine">
      <attribute name="title">
       <string>Звершение линии</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>481</width>
         <height>102</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QRadioButton" name="doubleClickLMB_2">
          <property name="text">
           <string>По двойному клику ЛКМ</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="holdingCtrl_2">
          <property name="text">
           <string>Поставить последнюю точку с зажатым Ctrl</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="keyC_2">
          <property name="text">
           <string>По клавише C</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
       <property name="centerButtons">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
